src/net/mod.rs

```diff
@@ -593,9 +593,9 @@ impl PeerAddress {
     pub fn is_in_private_range(&self) -> bool {
         if self.is_ipv4() {
             // 10.0.0.0/8, 172.16.0.0/12, or 192.168.0.0/16
-            self.0[13] == 10
-                || (self.0[13] == 172 && self.0[14] >= 16 && self.0[14] <= 31)
-                || (self.0[13] == 192 && self.0[14] == 168)
+            self.0[12] == 10
```

Comment:
Is this a regression, or fixes an existing bug?

Reply:
Yes, both.